### DISCUSSION TITLE
Is possible enable concurrent mode with emotion now?

### DISCUSSION THREAD

**Discussion by uonr** [vercel/next.js#31669] [Help]
I tried enable concurrent mode but browser shown 500 page and there is no error log in terminal or browser.

Is emotion not prepared yet? maybe this issue is related https://github.com/emotion-js/emotion/issues/2524

**Comment by sanjaiyan-dev** [comment]
By the [maintainer of Emotion CSS](https://github.com/reactwg/react-18/discussions/113#discussioncomment-1672153)  about React 18 support :)
